Technical Charter.md

```diff
@@ -0,0 +1,242 @@
+# Technical Charter (the “Charter”) for vis.gl a Series of LF Projects, LLC
+
+**Adopted ________**
+
+This charter (the “Charter”) sets forth the responsibilities and procedures for
+technical contribution to, and oversight of, the **vis.gl** project,
+which has been established as **vis.gl** a Series of LF Projects, LLC
+(the “Project”). LF Projects, LLC (“LF Projects”) is a Delaware series limited
+liability company. All contributors (including committers, maintainers, and
+other technical positions) and other participants in the Project (collectively,
+“Collaborators”) must comply with the terms of this Charter.
+
+1.  Mission and Scope of the Project
+
+    1.  The mission of the Project is to provide a comprehensive, open-source solution for GPU-powered visualizations of large-scale datasets, accelerating software development and research in urban mobility, health, communication and other demains.

```

Comment:
@tgorkin @ibgreen @tsherif

Reply:
Yes, I think this looks good, this scope should accommodate all current planned development, as it does not tie vis.gl to any specific programming language or even to a geospatial focus (though I would argue that the urban computing part of implies that).

Reply:
"domains".

I'd actually emphasize the geospatial, without excluding others, since that's the domain in which vis.gl really stands out. Maybe something like "of large-scale datasets, optimized in particular for geospatial data visualizations." 

I might drop the clause starting with "accelerating software development..." It feels like it's just "any domain that benefits from data visualization"?

Reply:
I do want to somehow call out to UCF's mission statement, "to enable widespread adoption and help accelerate development of open source urban computing and a surrounding ecosystem."